.github/workflows/install.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: erdpy-build
+name: mxpy-build
```

Comment:
In the workflows files, we can simply remove `mxpy-` as well, and only leave `build:`  etc.